project-directory.md

```diff
@@ -64,26 +64,26 @@ These are all the repos found in the IPFS Github organization. If you still can'
 
 ### In Development
 
-* [node-ipfs](https://github.com/ipfs/node-ipfs)  
+* [go-ipld](https://github.com/ipfs/go-ipld)  
+    This is the Go implementation of the IPLD spec.
+
+* [js-ipfs](https://github.com/ipfs/js-ipfs)  
     An IPFS implementation in Node.js.
 
 * [py-ipfs](https://github.com/ipfs/py-ipfs)  
-    python implementation of IPFS.
-
-* [go-ipld](https://github.com/ipfs/go-ipld)  
-    This is the Go implementation of the IPLD spec.
+    Python implementation of IPFS.
 
 ### APIs
 
-* [node-ipfs-api](https://github.com/ipfs/node-ipfs-api)  
-    A node client library for the IPFS API.
-
 * [go-ipfs-api](https://github.com/ipfs/go-ipfs-api)  
     An unofficial go interface to IPFS's http API.
 
 * [java-ipfs-api](https://github.com/ipfs/java-ipfs-api)  
     A Java implementation of the HTTP IPFS API.
```

Comment:
I would say that, the library is the 'client' and implementation is the 'language'. So something like, `A client library for the IPFS HTTP API, implemented in <language>`